server/api/lib/tags/index.js

```diff
@@ -0,0 +1,62 @@
+ const { tags, tagDataRules } = require('../../constants');
+ const PlanAreaChanges = require('../../../api/model/plan_area_changes');
+
+ const  generateTagsForPlan = async (planId) => {
+    let planTags = [];
+    let dataRules = [];
```

Comment:
please change this let to const

Reply:
but I add things to it, so it's not a const...